migrations/alembic.ini

```diff
@@ -0,0 +1,45 @@
+# A generic, single database configuration.
```

Comment:
Just to correct @andela-dmigwi, migrations are intended to be pushed to github because they form part of the codebase. This link explains more:
http://stackoverflow.com/questions/28035119/should-i-be-adding-the-django-migration-files-in-the-gitignore-file